ouroboros-consensus/test-consensus/Test/Dynamic/Praos.hs

```diff
@@ -95,10 +95,15 @@ prop_simple_praos_convergence params numCoreNodes numSlots =
             -> Map NodeId (Chain (Block DemoPraos))
             -> Property
     isValid nodeIds final = counterexample (show final) $
-      let schedule = leaderScheduleFromTrace numSlots final
+      -- Oh dear, oh dear. This node config isn't used except in the RealPBFT
```

Comment:
This isn't a huge disaster, but I feel that we should be able to get that node config somewhere. We should (briefly) try.